src/tests/ftest/util/file_count_test_base.py

```diff
@@ -33,13 +33,14 @@ def get_rf(oclass):
 
 
 # pylint: disable=attribute-defined-outside-init
-class FileCountTestBase(MdtestBase, IorTestBase):
+class FileCountTestBase(IorTestBase, MdtestBase):
     # pylint: disable=too-many-ancestors
     """Test class Description: Runs IOR and MDTEST to create specified number of files.
 
     :avocado: recursive
     """
 
+
```

Comment:
The pydocstyle recommendation is single-spacing between methods inside a class and double-spacing between classes or standalone methods in a file.
```suggestion
```